internal/pkg/state/fixtures.go

```diff
@@ -1,28 +1,38 @@
 package state
 
 import (
+	"fmt"
 	"strings"
 
 	"github.com/keboola/keboola-as-code/internal/pkg/fixtures"
+	"github.com/keboola/keboola-as-code/internal/pkg/model"
+	"github.com/keboola/keboola-as-code/internal/pkg/testproject"
 	"github.com/keboola/keboola-as-code/internal/pkg/utils"
 )
 
 // NewProjectSnapshot - to validate final project state in tests.
-func NewProjectSnapshot(s *State) (*fixtures.ProjectSnapshot, error) {
+func NewProjectSnapshot(s *State, testProject *testproject.Project) (*fixtures.ProjectSnapshot, error) {
 	project := &fixtures.ProjectSnapshot{}
 
-	branches := make(map[string]*fixtures.BranchConfigs)
+	defaultBranchId := -1
```

Comment:
`testproject.Project` obsahuje `defaultBranch`, staci tam pridat metodu `DefaultBranch`, ktora ti ju vrati, ... bude to trocha prehladnejsie.